packages/algob/src/builtin-tasks/run.ts

```diff
@@ -43,7 +43,7 @@ export function splitAfter (
   return scriptsFromScriptsDir.splice(0, scriptsFromScriptsDir.length);
 }
 
-function loadCheckpointsIntoCPData (cpData: CheckpointRepo, scriptPaths: string[]): CheckpointRepo {
+export function loadCheckpointsIntoCPData (cpData: CheckpointRepo, scriptPaths: string[]): CheckpointRepo {
```

Comment:
Let's move this function to `src/lib` or `src/internal` (depending if it makes sens for a user or plugin to use it, if it's used only internally then it should go to internal). builting-tasks should be simple, and if there is a function used in multiple tasks, then it shouldn't be in `buildin-tasks`.

Reply:
Ok, good point. Moving to `src/lib`.